alignak_backend/app.py

```diff
@@ -229,6 +229,47 @@ def pre_get(resource, user_request, lookup):
                                        {'_realm': {'$in': resources_get_custom[resource]}}]}]
 
 
+# Hosts / services
+def on_return_host(response):
+    """
+    Hook before getting some hosts
+
+    :param items: realm fields
+    :type items: dict
+    :return: None
+    """
+    if '_items' in response:
+        ls_drv = current_app.data.driver.db['livestate']
+        for item in response['_items']:
+            ls = ls_drv.find_one({'host': item['_id']})
+            if not ls:
+                continue
+
+            for k in ls:
+                if k not in ['_id', '_etag', '_created', '_updated', '_realm', 'name', 'host']:
+                    item[k] = ls[k]
```

Comment:
I prefer put the fields in item['livestate'] to separate config fields of host/service to the state of host/service